solution/backend/regulations/templates/regulations/base.html

```diff
@@ -53,10 +53,6 @@
 
                 {% block body %}{% endblock %}
 
-                {% block login_indicator %}
-                    {% include "regulations/partials/login_indicator.html" %}
-                {% endblock %}
-
```

Comment:
login indicator and logout button at bottom right corner of screen has been removed, never to return